application/classes/Ushahidi/Listener/IntercomAdminListener.php

```diff
@@ -0,0 +1,74 @@
+<?php defined('SYSPATH') or die('No direct script access');
+
+/**
+ * Ushahidi Intercom Listener
+ *
+ * Listens for new posts that are added to a set
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Application
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+use League\Event\AbstractListener;
+use League\Event\EventInterface;
+
+use Intercom\IntercomClient;
+
+use GuzzleHttp\Exception\ClientException;
+
+class Ushahidi_Listener_IntercomAdminListener extends AbstractListener
+{
+
+  public function handle(EventInterface $event, $user = null)
+  {
+
+    $config = service('repository.config')->get('thirdparty');
```

Comment:
I don't think you need to use the repo here, just `service('thirdparty.intercomapptoken')`
The repo is only really necessary for things that are use editable